milestoneupdater/config.json

```diff
@@ -31,22 +31,13 @@
     "user_saml"
   ],
   "dueDates": {
-    "Nextcloud 12.0.12": "2018-10-11",
-    "Nextcloud 13.0.7": "2018-10-11",
-    "Nextcloud 14.0.1": "2018-10-11",
-    "Nextcloud 15": "2018-12-06"
```

Comment:
Please don't delete due dates for upcoming releases. It would make it harder to add the existing milestones to new repos ;) Let me fix that.